app/tests/unit/lib/theme/widgets/ContextualHelp.test.tsx

```diff
@@ -0,0 +1,36 @@
+import { render, screen } from "@testing-library/react";
+import { faExclamationCircle } from "@fortawesome/free-solid-svg-icons";
+import ContextualHelp from "lib/theme/widgets/ContextualHelp";
+
+describe("ContextualHelp component", () => {
+  const titleMock = "Tooltip Title";
```

Comment:
If we move away from the word "title", could change to whatever the replacement is here too